tests/test_get_replica_connections.py

```
@@ -0,0 +1,99 @@
+
+def sentinel_client():
+    client = FalkorDB(
+        host='localhost',
+        port=26379
+    )
+    return client
+
+def delete_replicas(client,mode):
+    try:
+        result = client.get_replica_connections()
+    except Exception as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)